src/content/reference/feature-flags.mdx

```diff
@@ -13,18 +13,21 @@ Admin variables are equivalent to defining that variable on every developer's ma
+| OKTETO_USE_STATIC_KUBETOKEN                      | Use a fixed token rather than a dynamic token for interactions with the Kubernetes API.                                                                                                                                         | `false`                |
+| OKTETO_AUTOGENERATE_STIGNORE                     | If true, generates the `.stignore` file when running `okteto up`.                                                                                                                                                               | `false`                |
+| OKTETO_AUTODEPLOY                                | If set, forces the deployment of the development environment on `okteto up`.                                                                                                                                                    | `false`                |
+| OKTETO_COMPOSE_UPDATE_STRATEGY                   | Defines the update strategy that the compose must translate (it can be one of: `rolling`/`recreate`/`on-delete`).                                                                                                               | N/A                    |
+| OKTETO_DISABLE_SPINNER                           | If set, disables the spinner rotation.                                                                                                                                                                                          | `false`                |
+| OKTETO_SYNCTHING_VERSION                         | Specifies the Syncthing version the CLI must use.                                                                                                                                                                               | N/A                    |
+| OKTETO_BUILDKIT_WAIT_TIMEOUT                     | Specifies the maximum duration to wait for BuildKit to become available.                                                                                                                                                        | `10m`                  |
+| OKTETO_RETRY_BUILDKIT_INTERVAL                   | Specifies the interval between successive checks of BuildKit's availability status.                                                                                                                                             | `5s`                   |
+| OKTETO_MAX_RETRIES_FOR_BUILDKIT_TRANSIENT_ERRORS | Specifies the maximum number of retry attempts for build processes that fail due to transient (temporary) BuildKit errors.                                                                                                      | `3`                    |
```

Comment:
If we have time, I would rename this to `OKTETO_BUILDKIT_MAX_RETRIES_FOR_TRANSIENT_ERRORS`, that way buildkit variables are together is everything is sorted alphabetically